doc/language.md

```diff
@@ -37,21 +39,14 @@ the `select` is used, it will only keep selected columns in the output.
 ```javascript
 {
     name: 'select',
-    column: 'my-column'
+    column: 'my-column' -> should accept expressions like "*" and "(my_colmun * my_other_column) / 100"
+    as: 'new-column-name'
 }
 ```
 
 ### 'rename' step
 
-Rename a column.,
-
-```javascript
-{
-    name: 'filter',
-    oldname: 'old-column-name',
-    newname: 'new-column-name'
-}
-```
+I don't think this step is needed with a more "complete" `select`.
 
```

Comment:
True but that leads to UI / UX questions on query edition afterwards: *rename* seems easy to understand and edit. A *select* statement will probably merged with more complex operations than the rename itself and I fear it will be harder to edit or render it nicely. :thinking: